rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize.cpp

```diff
@@ -46,16 +42,15 @@ void ResizeNode::create_node() {
     width_status = vxAddArrayItems(_dst_roi_width, _batch_size, dst_roi_width.data(), sizeof(vx_uint32));
     height_status = vxAddArrayItems(_dst_roi_height, _batch_size, dst_roi_height.data(), sizeof(vx_uint32));
      if(width_status != 0 || height_status != 0)
-        THROW(" vxAddArrayItems failed in the resize (vxExtrppNode_ResizebatchPD) node: "+ TOSTR(width_status) + "  "+ TOSTR(height_status))
-#if ENABLE_OPENCL
-    // fall back to batch_pd version since there is no tensor implementation for OPENCL    
-    // _node = vxExtrppNode_ResizebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _batch_size);
-#else
-   // _node = vxExtrppNode_Resizetensor(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _interpolation_type, _batch_size);
-#endif
+        THROW("vxAddArrayItems failed in the resize (vxRppResize) node: " + TOSTR(width_status) + "  " + TOSTR(height_status));
+
+    vx_scalar interpolation_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()),VX_TYPE_INT32,&_interpolation_type);
+   _node = vxRppResize(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _dst_roi_width, 
```

Comment:
Please check if we can use dst_tensor_roi instead of _dst_roi_width & _dst_roi_height here

Reply:
Not required for now